lib/db_connection/connection.ex

```diff
@@ -333,11 +333,11 @@ defmodule DBConnection.Connection do
   def handle_event(:info, msg, :no_state, %{mod: mod, state: state} = s) do
     if function_exported?(mod, :handle_info, 2) do
       case apply(mod, :handle_info, [msg, state]) do
```

Comment:
I'm still passing the state to the callback since the monitor ref the monitor ref / socket ref would be there, and they are probably needed to match on the incoming message.